### ISSUE TITLE
Grincher is being a mean one (Still entering lobbies and getting kicked right after)

### ISSUE THREAD

**Issue by Fraisk** [Vildravn/LobbyLifeguard#1]
Despite having this mod, with the banlist, grincher can still enter lobbies, then getting kicked again.

![image](https://github.com/user-attachments/assets/3e8afa14-6ba8-4065-af4e-915f55567bf2)

I swear this is the only video I have showing this issue in action

https://medal.tv/games/webfishing/clips/jeNc7GmVkoEJX9BFy?invite=cr-MSxyUUIsMjgzOTc5MTA5LA

**Comment by Seltraeh** [comment]
Agreed, now there is a guy with steam ID "notb11" entering lobbies but never fully connecting and it's annoying.
